src/core/components/auth/auths.jsx

```diff
@@ -80,12 +80,13 @@ export default class Auths extends React.Component {
                   />
               }).toArray()
             }
-            <div className="auth-btn-wrapper">
+            <div className="auth-btn-wrapper sui-btn-wrapper">
               {
-                nonOauthDefinitions.size === authorizedAuth.size ? <Button className="btn modal-btn auth" onClick={ this.logoutClick }>Logout</Button>
-              : <Button type="submit" className="btn modal-btn auth authorize">Authorize</Button>
+                nonOauthDefinitions.size === authorizedAuth.size
+                  ? <Button className="sui-btn sui-btn--tertiary-lt modal-btn auth" onClick={ this.logoutClick } ><span>Logout</span></Button>
+                  : <Button type="submit" className="sui-btn sui-btn--primary modal-btn auth authorize"><span>Authorize</span></Button>
               }
-              <Button className="btn modal-btn auth btn-done" onClick={ this.close }>Close</Button>
+              <Button className="sui-btn sui-btn--secondary modal-btn auth btn-done" onClick={ this.close }><span>Close</span></Button>
```

Comment:
**Question**: should `btn-done` be prefixed with `sui` and/or should `done` be a modifier of `btn`?

Reply:
Possibly... I'd say more likely that it is no longer needed. Will check an tidy up as necessary :)